DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/TutorialViewModelTests.cs

```diff
@@ -160,5 +160,66 @@ public void Tutorial_should_not_recommend_a_tutorial_when_score_is_max()
             // Then
             tutorialViewModel.TutorialRecommendation.Should().Be("optional");
         }
+
+        [Test]
+        public void Tutorial_parsed_objectives_are_null_when_objectives_are_null()

```

Comment:
This test name doesn't quite read like a sentance. Maybe something like `Tutorial_handles_null_objectives`? 🙂

Reply:
That's much better, thank you! Changed in 3ff95e1 🙂